test/module/index.js

```diff
@@ -89,7 +89,7 @@ describe('Module Tests', function() {
 
   it('should be able to run lighthouse with just a url and options', function() {
     const lighthouse = require('../..');
-    return lighthouse(VALID_TEST_URL, {})
+    return lighthouse(VALID_TEST_URL, {fast: true})
```

Comment:
This test only asserts there was truthy output. Unless it's looking for something more specific, I don't think we need to run everything. That's technically what the smoketest does.